owasp-top10-2017-apps/a6/stegonography/app/index.js

```diff
@@ -48,7 +48,7 @@ MongoClient.connect(url, function(err, db) {
 MongoClient.connect(url, function(err, db) {
     if (err) throw err;
     var dbo = db.db("stego");
-    var myobj = { username: "admin", password: "admin" };
+    var myobj = { username: "Dino01", password: "Dino20878"};
```

Comment:
What do you think of maybe using environment variables to get this `username` and `password`? By doing this, they will no longer be hardcoded in the application's source code. 😁